src/test/java/net/datafaker/script/ProvidersDocsGenerator.java

```diff
@@ -0,0 +1,197 @@
+package net.datafaker.script;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ast.comments.CommentsCollection;
+import com.github.javaparser.ast.comments.JavadocComment;
+import com.google.common.base.Strings;
+import net.datafaker.AbstractProvider;
+import org.reflections.Reflections;
+
+import java.io.*;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.reflections.scanners.Scanners.SubTypes;
+
+public class ProvidersDocsGenerator {
+    private static final Pattern pattern = Pattern.compile("\\d\\.\\d\\.\\d");
```

Comment:
```suggestion
    private static final Pattern pattern = Pattern.compile("\\d\\.\\d+\\.\\d");
```
otherwise it sill stop working starting since version 1.10.1